deploy-lighthouse-reth.py

```
@@ -0,0 +1,891 @@
+                    if chunk:
+                        t.update(len(chunk))
+                        f.write(chunk)
+            t.close()
+            print(f">> Successfully downloaded: {asset['name']}")
+
+        except requests.exceptions.RequestException as e:
+            print(f"Error: Unable to download file. Try again later. {e}")
+            exit(1)
+
+        # Extract the binary to the home folder
+        with tarfile.open(tar_filename, "r:gz") as tar:
```

Ruff (0.8.2)

`tar_filename` may be undefined, or defined from star imports

(F405)